analyses/copy_number_consensus_call/merge_cnv_file.R

```diff
@@ -0,0 +1,16 @@
+library(tidyverse)
+
+# set directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+cnv_gatk_dir <- file.path(root_dir, "analyses", "copy_number_consensus_call", "results")
+cnv_manta_dir <- file.path(root_dir, "analyses", "copy_number_consensus_call_manta", "results")
+output_file <- file.path(cnv_gatk_dir,  "cnv-consensus_full.seg.gz")
```

Comment:
Let's leave the name of this what is currently being released in `/data` since many modules depend on it. I would prefer if we updated the filenames of individual modules instead, so there is no confusion. You can also do that within this PR as well. So, let's do something like this:
for manta module:
 `cnv-consensus-freec-ckit-manta.seg.gz`
for gatk module:
 `cnv-consensus-freec-ckit-gatk.seg.gz`

then final filename for release:
 `cnv-consensus.seg.gz`

Would that work?